### PR TITLE
Date news card

### PR DESCRIPTION
**Pull request by martypowell**
Addresses #109

### CODE REVIEW

**Comment by martypowell** [comment]
I didn't add a test file or usage for the component, adding now.

**Comment by sdurphy** [comment]
it looks like at around 600px screen width, .dg_date-news-card__headline sticks out of the container... screenshot attached.
Screen Shot 2019-09-12 at 9 18 16 AM

**Review by sdurphy — CHANGES_REQUESTED** [review]
it looks like at around 600px screen width, .dg_date-news-card__headline sticks out of the container.

**Comment by martypowell** [comment]
Added tests, adding html next.

**Comment by martypowell** [comment]
added snippets. @danfox01 @jdomasky @sdurphy  this is ready for testing. This component includes all states right now, but once this is confirmed, I think we should remove the state examples in favor of just showing a default and dark example.

What do you guys think?

**Comment by danfox01** [comment]
> added snippets. @danfox01 @jdomasky @sdurphy this is ready for testing. This component includes all states right now, but once this is confirmed, I think we should remove the state examples in favor of just showing a default and dark example.
> 
> What do you guys think?

@martypowell agreed.

@jdomasky can you please test? Let me know if you need links/additional info.

**Comment by martypowell** [comment]
> it looks like at around 600px screen width, .dg_date-news-card__headline sticks out of the container... screenshot attached.
> Screen Shot 2019-09-12 at 9 18 16 AM

@sdurphy this is a result of the system layout, not the comopnet, see this screenshot, where i removed the sidebar:

image

**Comment by jdomasky** [comment]
@danfox01 @martypowell I'll begin testing for this component shortly.

**Comment by martypowell** [comment]
@danfox01 i think maybe i got the markup wrong here, and it should be an [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)? 

Also thinking for the date, maybe we need a screen-reader veriosn, since our date is decoration-y?

**Comment by danfox01** [comment]
@martypowell `<article> `seems like it would make most sense. I'm not sure about the accessibility on this one. We can look into it.

Are you pushing more changes or is @jdomasky good to test?

**Comment by martypowell** [comment]
> @martypowell
> 
> seems like it would make most sense. I'm not sure about the accessibility on this one. We can look into it.
> Are you pushing more changes or is @jdomasky good to test?

We can test. @danfox01 @jdomasky

**Comment by jdomasky** [comment]
@martypowell @danfox01 Testing is underway.

File: src/components/basic/DateNewsCard.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+### Date News Card
+
+#### Default State
+```jsx
+<div style={{ padding: '10px'}} className="dg_section">
```
Comment (theilman7) [inline src/components/basic/DateNewsCard.md:4]: Can the inline style included here be migrated to the style sheet? Disregard if this was setup for display purposes only.

**Comment by jdomasky** [comment]
@martypowell @sdurphy First round of testing is finished. Emailing results to @danfox01.

File: src/components/basic/DateNewsCard.md (hunk 1 above)
Comment (martypowell) [reply src/components/basic/DateNewsCard.md:4]: display purposes only @theilman7 but i do think that once we addressed the styles sheet styles, this should be addressed.

**Comment by danfox01** [comment]
Test results per @jdomasky, and triaged by @sdurphy and @martypowell:

- Disabled on dark background is incorrect.
- Month abbreviation in date should be call caps.
- Article headline should be title case.
- Spacing between date and headline needs to be adjusted to match mockup.
- Adjust padding around snippet to match mockup.
- Border separator above teaser should be 1px, not 2px.
- Border peparator position should be centered vertically between tghe elements above and below. (Padding fix will probably correct this.)
- Border color in hovered and disabled states should be `white`, not `gray-light`.

**Comment by martypowell** [comment]
> Test results per @jdomasky, and triaged by @sdurphy and @martypowell:
> 
> * Disabled on dark background is incorrect.
> * Month abbreviation in date should be call caps.
> * Article headline should be title case.
> * Spacing between date and headline needs to be adjusted to match mockup.
> * Adjust padding around snippet to match mockup.
> * Border separator above teaser should be 1px, not 2px.
> * Border peparator position should be centered vertically between tghe elements above and below. (Padding fix will probably correct this.)
> * Border color in hovered and disabled states should be `white`, not `gray-light`.

@danfox01 addressed.

@jdomasky ready for another look. Any of the spacing issues should be in 5px increments, and may be slightly different than the xd document.

**Comment by jdomasky** [comment]
@martypowell @danfox01 Update: Reviewing now.

**Comment by jdomasky** [comment]
@martypowell @danfox01 All of the testing issues have been resolved. This component is good to go.

**Comment by martypowell** [comment]
@danfox01 @sdurphy this is ready for a rubber stamp. I removed the state examples as we talked about earlier today.

**Review by danfox01 — APPROVED** [review]
Stamp

**Pull request merged**